### PR TITLE
Document the TABLE statement

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Add initial documentation for the `TABLE` statement.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### Open questions

- This statement also works in PostgreSQL (tested on 13.2) and it may conform to the ISO SQL standards. Should we document that as well?
- I don't think the EBNF is complete yet ( test on https://kennytm.github.io/website-docs/dist/ if you make any changes)

### CODE REVIEW

File: sql-statements/sql-statement-table.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: TABLE | TiDB SQL Statement Reference
+summary: An overview of the usage of TABLE for the TiDB database.
+aliases: ['/docs/dev/sql-statements/sql-statement-table/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-table.md:4]:
```suggestion
```

**Comment by TomShawn** [comment]
@dveeden Please add this document to `TOC.md` (document menu), thanks~

File: sql-statements/sql-statement-table.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+3 rows in set (0.01 sec)
+
+tidb 5.7.25-TiDB-v5.0.0 > TABLE t1 LIMIT 1;
++----+
+| id |
++----+
+|  1 |
++----+
+1 row in set (0.01 sec)
+```
```
Comment (morgo) [inline sql-statements/sql-statement-table.md:55]: Our new code standard is to include the SQL as copyable, then the output. i.e. (ignore the \ character - I can't seem to get the suggestion working):

```suggestion

{{< copyable "sql" >}}

\```sql
CREATE TABLE t1(id INT PRIMARY KEY);
INSERT INTO t1 VALUES (1),(2),(3);
TABLE t1;
TABLE t1 ORDER BY id DESC;
TABLE t1 LIMIT 1;
SELECT * FROM t1;
\```

\```sql
Query OK, 0 rows affected (0.31 sec)

Query OK, 3 rows affected (0.06 sec)
Records: 3  Duplicates: 0  Warnings: 0

+----+
| id |
+----+
|  1 |
|  2 |
|  3 |
+----+
3 rows in set (0.01 sec)

+----+
| id |
+----+
|  3 |
|  2 |
|  1 |
+----+
3 rows in set (0.01 sec)

+----+
| id |
+----+
|  1 |
+----+
1 row in set (0.01 sec)

+----+
| id |
+----+
|  1 |
|  2 |
|  3 |
+----+
3 rows in set (0.01 sec)

\```
```

File: sql-statements/sql-statement-table.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## MySQL compatibility
+
+The `TABLE` statement was introduced in MySQL 8.0.19.
+
+## See also
+
+- [TABLE statements in MySQL](https://dev.mysql.com/doc/refman/8.0/en/table.html)
```
Comment (morgo) [inline sql-statements/sql-statement-table.md:63]: See also `SELECT` (and in the example I've shown a select equivalent).

**Review by morgo — COMMENTED** [review]
LGTM, minor style nits

**Comment by dveeden** [comment]
@TomShawn @morgo I've incorporated your suggestions.

File: sql-statements/sql-statement-table.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE t1(id INT PRIMARY KEY);
+```
+
+```sql
+Query OK, 0 rows affected (0.31 sec)
+```
+
```
Comment (TomShawn) [inline sql-statements/sql-statement-table.md:28]:
```suggestion

{{< copyable "sql" >}}

```

File: sql-statements/sql-statement-table.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+```sql
+INSERT INTO t1 VALUES (1),(2),(3);
+```
+
+```sql
+Query OK, 3 rows affected (0.06 sec)
+Records: 3  Duplicates: 0  Warnings: 0
+```
+
```
Comment (TomShawn) [inline sql-statements/sql-statement-table.md:37]:
```suggestion

{{< copyable "sql" >}}

```

File: sql-statements/sql-statement-table.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
++----+
+| id |
++----+
+|  1 |
+|  2 |
+|  3 |
++----+
+3 rows in set (0.01 sec)
+```
+
```
Comment (TomShawn) [inline sql-statements/sql-statement-table.md:52]:
```suggestion

{{< copyable "sql" >}}

```

File: sql-statements/sql-statement-table.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
++----+
+| id |
++----+
+|  3 |
+|  2 |
+|  1 |
++----+
+3 rows in set (0.01 sec)
+```
+
```
Comment (TomShawn) [inline sql-statements/sql-statement-table.md:67]:
```suggestion

{{< copyable "sql" >}}

```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**